web/src/components/roadmap/SidebarCardsList.tsx

```diff
@@ -0,0 +1,154 @@
+import { OpportunityCard } from "@/components/dashboard/OpportunityCard";
+import { Icon } from "@/components/njwds/Icon";
+import { RoadmapSidebarCard } from "@/components/roadmap/RoadmapSidebarCard";
+import { useRoadmapSidebarCards } from "@/lib/data-hooks/useRoadmapSidebarCards";
+import { useUserData } from "@/lib/data-hooks/useUserData";
+import { filterCertifications } from "@/lib/domain-logic/filterCertifications";
+import { sortCertifications } from "@/lib/domain-logic/sortCertifications";
+import { Certification, Funding, SidebarCardContent } from "@/lib/types/types";
+import { templateEval } from "@/lib/utils/helpers";
+import Config from "@businessnjgovnavigator/content/fieldConfig/config.json";
+import { LookupOperatingPhaseById } from "@businessnjgovnavigator/shared/operatingPhase";
+import { Accordion, AccordionDetails, AccordionSummary } from "@mui/material";
+import { ReactElement, useEffect, useState } from "react";
+
+interface Props {
+  sidebarDisplayContent: Record<string, SidebarCardContent>;
+  certifications: Certification[];
+  fundings: Funding[];
+}
+
+export const SidebarCardsList = (props: Props): ReactElement => {
+  const { userData } = useUserData();
+  const { hideCard } = useRoadmapSidebarCards();
+  const [hiddenAccordionIsOpen, setHiddenAccordionIsOpen] = useState<boolean>(false);
+
+  //   const filteredSortedFundings = userData ? sortFundings(filterFundings(props.fundings, userData)) : [];
+  const filteredSortedCertifications = userData
+    ? sortCertifications(filterCertifications(props.certifications, userData))
+    : [];
+  //   const nonHiddenSortedFundings = filteredSortedFundings.filter(
+  //     (it) => !userData?.preferences.hiddenFundingIds.includes(it.id)
+  //   );
+  const nonHiddenSortedCertifications = filteredSortedCertifications.filter(
```

Comment:
instead of "nonHidden" it might be clearer to use the positive case instead of the negative case. "shown" or "visible" maybe?